tsconfig.strict.json

```diff
@@ -51,6 +51,8 @@
     "lib/util/host-rules.ts",
     "lib/util/html.ts",
     "lib/util/http/**/.ts",
+    "lib/util/lazy.ts",
+    "lib/util/lazy.spec.ts",
```

Comment:
```suggestion
    "lib/util/**/*.ts",
```
Less code 😉

Reply:
Okay, let's do this as the separate task. We still need to exclude some util files, though I need some time to identify them